requirements.txt

```diff
@@ -0,0 +1 @@
+Adafruit-Blinka
```

Comment:
Like the other two, Blinka isn't needed since no PyPI.

Reply:
Done